README.md

```diff
@@ -142,8 +151,9 @@ $ sfn print --file lambda_test
     "MyHandlerLambdaFunction": {
       "Type": "AWS::Lambda::Function",
       "Properties": {
-        "Handler": "python2.7",
-        "FunctionName": "my_handler",
+        "Handler": "index.my_handler",
```

Comment:
According to AWS, when using ZipFile the handler must be specified as `index.function_name`. http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html#cfn-lambda-function-handler